src/main/java/com/marklogic/contentpump/SplitDelimitedTextReader.java

```diff
@@ -45,6 +45,7 @@ public class SplitDelimitedTextReader<VALUEIN> extends
     DelimitedTextReader<VALUEIN> {
     public static final Log LOG = LogFactory
         .getLog(SplitDelimitedTextReader.class);
+    private static final String INVALID_TOKEN_DELIMITER_ERROR = "Invalid character between encapsulated token and delimiter";
```

Comment:
Sorry overlooked the hierarchy of these two classes, the string should be protected so that it can be shared to the child class?

Reply:
I agree, the right way to do it. Resolved in the last commit.